modules/openxr/scene/openxr_hand.cpp

```diff
@@ -116,6 +133,16 @@ void OpenXRHand::_set_motion_range() {
 	hand_tracking_ext->set_motion_range(OpenXRHandTrackingExtension::HandTrackedHands(hand), xr_motion_range);
 }
 
+void OpenXRHand::set_skeleton_rig(const SkeletonRig p_skeleton_rig) {
```

Comment:
```suggestion
void OpenXRHand::set_skeleton_rig(SkeletonRig p_skeleton_rig) {
```

Reply:
Cleaned up to remove the const on this (and the other two) simple-type setter.